aweadmin.py

```diff
@@ -53,6 +55,50 @@ def get_jobs(baseurl,token,status):
   dumpdata(data)
   return data['data']
 
+def get_queues(baseurl,token,status):
```

Comment:
Does this function do anything important?  I'm not familiar with the queue endpoint of AWE, and it's not used anywhere else.